source/speech/__init__.py

```diff
@@ -766,8 +769,25 @@ def _speakTextInfo_addMath(speechSequence, info, field):
 	except (NotImplementedError, LookupError):
 		return
 
-def speakTextInfo(info, useCache=True, formatConfig=None, unit=None, reason=controlTypes.REASON_QUERY, _prefixSpeechCommand=None, onlyInitialFields=False, suppressBlanks=False,priority=None):
+re_white_space = re.compile(r"(\s+|$)", re.DOTALL)
+
+class _TextCommand(str):

```

Comment:
NIT: Command's usually inherit from the BaseCommand class or a subclass. I'm concerned people will be confused with this classes name.

Reply:
The subclass is only there to distinguish text strings from strings that are created from control or format fields. DO you have a suggestion for an alternate name?